src/dmd/dsymbolsem.d

```diff
@@ -654,9 +654,8 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             {
                 if (global.params.vfield && dsym.storage_class & (STC.const_ | STC.immutable_) && dsym._init && !dsym._init.isVoidInitializer())
                 {
-                    const(char)* p = dsym.loc.toChars();
                     const(char)* s = (dsym.storage_class & STC.immutable_) ? "immutable" : "const";
-                    message("%s: %s.%s is %s field", p ? p : "", ad.toPrettyChars(), dsym.toChars(), s);
+                    message(dsym.loc, "`%s.%s` is `%s` field", ad.toPrettyChars(), dsym.toChars(), s);
```

Comment:
You need to update some tests for this e.g.

```
==============================
Test failed: 
expected:
----
compilable/sw_transition_field.d(15): sw_transition_field.S1.ix is immutable field
compilable/sw_transition_field.d(16): sw_transition_field.S1.cx is const field
compilable/sw_transition_field.d(21): sw_transition_field.S2!(immutable(int)).S2.f is immutable field
compilable/sw_transition_field.d(21): sw_transition_field.S2!(const(int)).S2.f is const field
----
actual:
----
compilable/sw_transition_field.d(15): `sw_transition_field.S1.ix` is `immutable` field
compilable/sw_transition_field.d(16): `sw_transition_field.S1.cx` is `const` field
compilable/sw_transition_field.d(21): `sw_transition_field.S2!(immutable(int)).S2.f` is `immutable` field
compilable/sw_transition_field.d(21): `sw_transition_field.S2!(const(int)).S2.f` is `const` field
----
```

Reply:
Right, always let the CI find this out so I can fix up later. ;-)

Should be fine now.